### PR TITLE
Installation specific documentation

### PR DESCRIPTION
**Pull request by Adron**
Added installation doc & call out in the getting-started.md/html file to the installation specific docs.

Per #1011

### CODE REVIEW

File: docs/content/getting-started.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This guide will show you
   * [how to manually setup Paket](getting-started.html#Manual-setup) in your .NET / mono solutions
   * and [how to use the automatic NuGet conversion](getting-started.html#Automatic-NuGet-conversion).
 
-<blockquote>The following guide is assuming you are using the paket.exe command line tool.
+<blockquote>The following guide is assuming you are using the paket.exe command line tool. For information on insalling the commmand line tool follow the instructions for your operating system for <a href="installation.html">installation</a>.
```
Comment (forki) [inline docs/content/getting-started.md]: There is a third missing in installing

File: docs/content/installation.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Paket Installation Options
+
+This guide will show you
+
+  * How to install Paket for Windows, Linux, or OS-X.
+  * How to insure `paket` is available via commmand line and other methods of use.
```
Comment (forki) [inline docs/content/installation.md]: Do you mean ensure?

File: docs/content/installation.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+<blockquote>The following guide provides the various installation methods for each of the three top developer operating systems.</blockquote>
+
+The first step for any operatinng system is to clone the [Paket repository]() locally.
+
+    git clone git@github.com:fsprojects/Paket.git
+
```
Comment (forki) [inline docs/content/installation.md]: Operating system has without the second n

**Comment by forki** [comment]
Very cool. That's super helpful

**Comment by Adron** [comment]
I think that gets everything. I reread again myself, nothing stuck out at me.

**Comment by forki** [comment]
fixed that typo. thx a lot

**Comment by forki** [comment]
I added a small paragraph for "install per repo" http://fsprojects.github.io/Paket/installation.html#Installation-per-repository - could you please review this? Thx

**Pull request merged**